README.md

```diff
@@ -38,3 +33,26 @@ the code passed back from WorkOS to grab the profile of the authenticated user t
 ```php
 $profile = (new \WorkOS\SSO())->getProfile($code);
 ```
+
+### Audit Trail
+Creating an Audit Trail event requires a descriptive action name and annotating the event with its CRUD identifier. The action name must contain an action category and an action name separated by a period, for example, user.login.
```

Comment:
Nit: Can you wrap user.login with ticks for Markdown rendering: `user.login`?

Reply:
Actually, this whole bit was leftover copy pasta from the docs that I wanted to get rid of. Gonna leave a quick example of audit trail here and if users want to know more, they can check out our docs site.